app/controllers/sub_category_quizzes_controller.rb

```diff
@@ -30,4 +55,13 @@ def answer
       render json: { error: result[:error], details: result[:details] }, status: result[:status]
     end
   end
+
+  private
+  def sub_category_quiz_params
+    params.require(:sub_category_quiz).permit(:sub_category_id, :quiz_question, :correct_answer_index, quiz_options: [])
+  end
+
+  def update_sub_category_quiz_params
+    params.require(:sub_category_quiz).permit(:sub_category_quiz_id, :sub_category_id, :quiz_question, :correct_answer_index, quiz_options: [])
+  end
```

Comment:
Do you need to strong params? I think it's better to merge extra params with strong params when needed